examples/sdk/auth-delegated/mobile/package.json

```diff
@@ -10,7 +10,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@dfns/sdk-react-native": "0.3.3",
+    "@dfns/sdk-react-native": "0.4.0-alpha.2",
```

Comment:
I this `alpha.2` expected for a dfns package?

Reply:
nope. let me fix that